planning/user_stories.md

```diff
@@ -4,10 +4,20 @@ Reference the Writing User Stories final project guide in the course portal for
 
 ## Outline User Roles
 
-[👉🏾👉🏾👉🏾 Include at least at least 1, but no more than 3, user roles.]
+- Fridge member: a user who wants to manage the food in the fridge
 
 ## Draft User Stories
 
-[👉🏾👉🏾👉🏾 Include at least at least 10 user stories in this format:]
-
-1. As a [user role], I want to [what], so that [why].
+1. As a member, I want to see what kinds of food are in the fridge, so that I can decide what to cook.
+2. As a member, I want to add food and update the count of each food, so that the updated information is shared.
+3. As a member, I want to know the expiration dates of each ingredient, so that I can use food before it expires.
+4. As a member, I want to mark missing ingredients to buy, so that a member knows what to buy.
+5. As a member, I want to create a fridge, so that I can start tracking food in the fridge.
+6. As a member, I want to invite or remove users from the fridge, so that the fridge info is shared only with the people who need the info.
+7. As a member, I want to get a list of items I need to buy, so that I don't need to browse the entire list of food in the fridge.
+8. As a member, I want to delete the fridge, so that unused fridges don't float around.
+9. As a member, I want to make the food item invisible when there is no food item anymore, so that the fridge info doesn't get cluttered.
+10. As a member, I want to keep the food data even after making the food item invisible, so that I can add the food item from the history list.
```

Comment:
I'm a bit confused here. Maybe I didn't understand correctly. So the invisible food is food with count=0, right? And the app should have a history list, and the history list is used to record invisable food items?

Reply:
I thought the food item history might be useful, but it would be a low priority. I removed the items for now.